### ISSUE TITLE
Not seeing the Resizing Behavior icon

### ISSUE THREAD

**Issue by markd2** [secondgear/Achieving-Zen-With-Auto-Layout#12]
Probably an #ilyxc moment - fig 16 references a "resizing behavior" tool, but looks like that got removed in newer Xcodes.

**Comment by justin** [comment]
Yeah, you're right. They replaced it with the Magic Stack Views button. I'll get this cleaned up.
